packages/storage/exp/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@firebase/storage/exp",
+  "description": "A tree-shakeable version of the Storage SDK",
+  "main": "../dist/exp/index.cjs.js",
+  "module": "../dist/exp/index.esm2017.js",
+  "browser": "../dist/exp/index.esm2017.js",
+  "typings": "../exp-types/index.d.ts",
```

Comment:
Should we auto-generate the typing file instead of maintaining it manually? As I mentioned in earlier comments, we can defined the public types in `storage-types` and make the exported functions use them.

Reply:
Pointed this at generated index.d.ts